packages/ui-patterns/InfoTooltip/InfoTooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import { Info } from 'lucide-react'
```

Comment:
are you meant to be using this instead of the SVG below? otherwise this import is not being used